rep-2005.rst

```diff
@@ -0,0 +1,237 @@
+  * `ros2/rcl_interfaces <https://github.com/ros2/rcl_interfaces>`_
+  * `ros2/test_interface_files <https://github.com/ros2/test_interface_files>`_
+  * `ros2/unique_identifier_msgs <https://github.com/ros2/unique_identifier_msgs>`_
+
+* RMW
+
+  * `ros2/rmw <https://github.com/ros2/rmw>`_
+  * `ros2/rmw_implementation <https://github.com/ros2/rmw_implementation>`_
+  * Connext (Connext itself is not open source)
+
+    * `ros2/rmw_connext <https://github.com/ros2/rmw_connext>`_
+    * `ros2/rosidl_typesupport_connext <https://github.com/ros2/rosidl_typesupport_connext>`_
+
+  * CycloneDDS
+
+    * `eclipse-cyclonedds/cyclonedds <https://github.com/eclipse-cyclonedds/cyclonedds>`_
+    * `ros2/rmw_cyclonedds <https://github.com/ros2/rmw_cyclonedds>`_
+
+  * FastRTPS
+
+    * `eProsima/Fast-CDR <https://github.com/eProsima/Fast-CDR>`_
+    * `eProsima/Fast-RTPS <https://github.com/eProsima/Fast-RTPS>`_
+    * `ros2/rmw_fastrtps <https://github.com/ros2/rmw_fastrtps>`_
+    * `ros2/rosidl_typesupport_fastrtps <https://github.com/ros2/rosidl_typesupport_fastrtps>`_
+
+  * OpenSplice
+
+    * `ros2/rmw_opensplice <https://github.com/ros2/rmw_opensplice>`_
+    * `ros2/rosidl_typesupport_opensplice <https://github.com/ros2/rosidl_typesupport_opensplice>`_
+
+* Client libraries
+
+  * `ros2/rcl <https://github.com/ros2/rcl>`_
+  * `ros2/rcl_logging <https://github.com/ros2/rcl_logging>`_
+  * `ros2/rclcpp <https://github.com/ros2/rclcpp>`_
+  * `ros2/rclpy <https://github.com/ros2/rclpy>`_
+
+* Orchestration
+
+  * `ros2/launch <https://github.com/ros2/launch>`_
+  * `ros2/launch_ros <https://github.com/ros2/launch_ros>`_
+
+* Features
+
+  * `ros/diagnostics <https://github.com/ros/diagnostics>`_
+  * `ros/xacro <https://github.com/ros/xacro>`_
+  * Sensor processing
+
+    * `ros-perception/image_common <https://github.com/ros-perception/image_common>`_
+    * `ros-perception/image_transport_plugins <https://github.com/ros-perception/image_transport_plugins>`_
+    * `ros-perception/perception_pcl <https://github.com/ros-perception/perception_pcl>`_
+    * `ros-perception/vision_opencv <https://github.com/ros-perception/vision_opencv>`_
+    * `ros-perception/laser_filters <https://github.com/ros-perception/laser_filters>`_
+    * `ros-perception/laser_geometry <https://github.com/ros-perception/laser_geometry>`_
+
+  * `ros-planning/navigation2 <https://github.com/ros-planning/navigation2>`_
+  * `ros-planning/moveit2 <https://github.com/ros-planning/moveit2>`_
+  * `ros-visualization/interactive_markers <https://github.com/ros-visualization/interactive_markers>`_
+  * `ros2/geometry2 <https://github.com/ros2/geometry2>`_
+  * `ros2/kdl_parser <https://github.com/ros2/kdl_parser>`_
+  * `ros2/message_filters <https://github.com/ros2/message_filters>`_
+  * `ros2/sros2 <https://github.com/ros2/sros2>`_
+  * `ros2/teleop_twist_joy <https://github.com/ros2/teleop_twist_joy>`_
```

Comment:
Sounds good to me. Just curious why the diff shows line 172 being touched?

Reply:
Because I don't know what I'm doing and made the suggestion starting with that line, which mean leaving that line alone and adding other lines under it :stuck_out_tongue: .

Reply:
Maybe GitHub just rendering it weirdly... I will apply it and check how it looks afterwards. Thanks.